zabbix/api.py

```diff
@@ -121,13 +121,22 @@ def do_request(self, method, params=None):
       z = ZabbixAPI()
       apiinfo = z.do_request('apiinfo.version')
     """
-    request_json = {
-      'jsonrpc':'2.0',
-      'method': method,
-      'params': params or {},
-      'auth': self.auth,
-      'id': '1',
-    }
+
+    if method = 'user.login':
```

Comment:
I tested the solution above and confirm that the change suggested by @blacked is working on Zabbix version 2.2.3